docs/eventing/brokers/broker-types/mt-channel-based-broker/README.md

```diff
@@ -0,0 +1,117 @@
+# Multi-tenant channel based Broker
+
+The Multi-tenant channel based Broker (MTChannelBasedBroker) uses [Channels](../../../channels) for event routing. It is shipped by default with Knative Eventing.
+Users should prefer native Broker implementations over the MTChannelBasedBroker and channel combo because it is usually more efficient. 
+## Prerequisites
+
+You need to have Knative Eventing installed.
+
+## Install a channel implementation
+
+As the MTChannelBasedBroker is based on Channels, you need to install a Channel implementation. Check out the [available Channels](../../../channels/channels-crds.md) for a (non-exhaustive) list of the available Channels for Knative Eventing.
+
+You can install e.g. the in-memory channel via:
+
+```bash
+kubectl apply -f {{ artifact(repo="eventing",file="in-memory-channel.yaml")}}
+```
```

Comment:
Make this a bullet in the prerequisites instead of a separate heading / section

Reply:
:+1: Updated